server/src/Emails.hs

```diff
@@ -153,6 +149,9 @@ send cfg email =
                     PasswordReset.getSuccess
                 OrderPlacedData parameters ->
                     OrderPlaced.get parameters
+                EmailVerificationData _ vCode -> 
+                    EmailVerification.get domainName (L.fromStrict vCode) 
```

Comment:
A note for the future.
We should make `domainName` configurable rather than hardcoded, because currently it doesn't support the staging use case.
I don't insist in changing this right now

Reply:
Perhaps, we can consider the `Host` header if it is present to determine the current domain

Reply:
Ah, oh, okay:(